Update INSTALLATION.md

spdegabrielle opened the pull request:
address issues in #253 and #254

BNAndras commented:
Sorry about the wait, I'm the new Racket maintainer, and I'll take a look at these old PRs. :)

BNAndras commented on docs/INSTALLATION.md:
````diff
@@ -1,36 +1,42 @@
-```
-pacman -S racket
-```
+You man need to prefix `sudo` to the start of the command to install to a location that requires adminstrator access.
````
```suggestion
You may need to prefix `sudo` to the start of the command to install to a location that requires administrator access.
```

BNAndras commented on docs/INSTALLATION.md:
````diff
@@ -1,36 +1,42 @@
-```
+You man need to prefix `sudo` to the start of the command to install to a location that requires adminstrator access.
+
+**Warning: Racket may also be available through your distribution's package manager, although it may be older than the latest Racket version.** While you may choose to do this later, as a learner you will have a much better experience if you stick to the current release available from https://download.racket-lang.org
````
```suggestion
**Warning: Racket may also be available through your distribution's package manager, although it may be older than the latest Racket version.**
While you may choose to do this later, as a learner you will have a much better experience if you stick to the current release available from https://download.racket-lang.org
```

BNAndras replied:
Exercism prefers one sentence per line.

spdegabrielle replied:
Ok

BNAndras commented on docs/INSTALLATION.md:
```diff
@@ -1,36 +1,42 @@
 
-### macOS official installer
-Note that on macOS, the official installer puts the files under `/Applications`, and as a result the `racket` and `raco` executables are not accessible on command-line terminals. You have to add the `bin` subdirectory below the Racket installation location (e.g., `/Applications/Racket v6.8/bin` for version 6.8) to the `$PATH`.  
+See [Command-Line Tools and Your Editor of Choice](https://docs.racket-lang.org/guide/other-editors.html) for an introduction to racket command line tool and using other editors.
```
```suggestion
See [Command-Line Tools and Your Editor of Choice](https://docs.racket-lang.org/guide/other-editors.html) for an introduction to Racket command line tools and using other editors.
```

spdegabrielle replied:
Ok

BNAndras commented on docs/INSTALLATION.md:
```diff
@@ -1,36 +1,42 @@
+* macOS
 
-Install via Homebrew:
+Please reference Racket's [Getting Started](http://docs.racket-lang.org/getting-started/) page for guidance materials.
```
The INSTALLATION.md is primarily intended to help students who want to work outside of the online Exercism editor. Resources about Racket that are relevant to both students working offline and online should go in RESOURCES.md or LEARNING.md. LEARNING.md should be specific to learning Racket, but RESOURCES.md is more general so things can be in both files. I think the links not about installing Racket can go sit in RESOURCES.md if they're not there already.

spdegabrielle replied:
The official getting started is specifically about getting and installing racket and setting up command line tools
I don’t see how this is not appropriate?
```suggestion
Please reference Racket's [Getting Started](http://docs.racket-lang.org/getting-started/) page for guidance materials.
```
![IMG_7418](https://github.com/exercism/racket/assets/108047/8a82ce80-05e7-43c5-9086-af2c39652c71)

BNAndras commented on docs/INSTALLATION.md:
```diff
@@ -1,36 +1,42 @@
+## About the macOS installer:
+
+If your machine uses the M1 or M2 series processors, select **Apple Silicon** in the platform dropdown menu before downloading.
+
```
Do we need these empty lines at the end? I think VSCode normally puts just one after when it formats Markdown.

spdegabrielle replied:
No. Empty lines can be removed

BNAndras commented:
I can deal with the merge conflict itself.

spdegabrielle commented:
Some notes

* removed Minimal Racket - it is not intended for beginners.
* changed download instructions to the official installers at https://download.racket-lang.org/ - brew and apt are not recommended for beginners and they often trail the official releases, leaving beginners in a situation where their install doesn’t match the docs at  https://docs.racket-lang.org/

thank you for taking responsibility for racket - I’m sure you will agree a resource that is maintained can be a great benefit - but an unmaintained resources can harm

BNAndras commented:
For what it's worth, I just made a PR to unpause community contributions for this repo so that should hopefully be less of a problem for contributors, especially first-time ones, in the future.

spdegabrielle commented:
> For what it's worth, I just made a PR to unpause community contributions for this repo so that should hopefully be less of a problem for contributors, especially first-time ones, in the future.

i’ll be honest and unmaintained resource with poor advice, is worse than no resource at all.

If you are serious about running a Racket track, I believe you should reach out to the record community directly at the Racket [Discourse](https://racket.discourse.group/) and [Discord](https://discord.gg/6Zq8sH5) ( these are the most active places for ‘Racketeers’) and explain why you believe volunteers should contribute to your site. Addressing why contributions were blocked for so long - and why the community should trust you not to do that again - would be a good start.
